<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HaircutMainWidj</class>
 <widget class="QMainWindow" name="HaircutMainWidj">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HaircutMainWidj</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_DownloadeFile">
      <property name="title">
       <string>Загрузка изображения</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="lineEdit_ImagePath">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pB_DownloadFIle">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="rB_gray">
         <property name="text">
          <string>GRAY</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="rB_red">
         <property name="text">
          <string>Red</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QRadioButton" name="rB_hue">
         <property name="text">
          <string>Hue</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="rB_green">
         <property name="text">
          <string>Green</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QRadioButton" name="rB_saturation">
         <property name="text">
          <string>Saturation</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QRadioButton" name="rB_blue">
         <property name="text">
          <string>Blue</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QRadioButton" name="rB_value">
         <property name="text">
          <string>Value</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>FILTERING and  PAINT</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="2">
        <widget class="QPushButton" name="pB_CreateMask">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>EDIT_MASK</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="6">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pB_filter">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>FILTER</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" rowspan="6">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QCheckBox" name="checkBox_dilate">
             <property name="text">
              <string>DILATE</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <weight>75</weight>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Power: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_dilatePower">
             <property name="maximum">
              <number>200</number>
             </property>
             <property name="value">
              <number>2</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="4">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Radius: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sB_inpaintRadius">
           <property name="value">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Treshold: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sB_treshold">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="lbl_noiseArea">
           <property name="text">
            <string>NOISE AREA: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sB_noiseArea">
           <property name="maximum">
            <number>10000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="pB_paint">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>PAINT (INPAINT OPEN CV)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QPushButton" name="pB_myPaint">
         <property name="text">
          <string>PAINT (MY METHOD)</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="lbl_hairWidth">
           <property name="text">
            <string>HAIR_WIDTH: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sB_hairWidth"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>545</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
